### PR TITLE
1.0.3 modified api

### PR DESCRIPTION
**Pull request by yimaixinchen**
### What is the purpose of the change
it can transform jersey to spring mvc api

### Brief change log
remove jersey api
add spring mvc api

### imcomplete part
but i can not transform requestbody in jersey to spring mvc .
just like FormDataMultiPart multiPartForm in RequestBody

### CODE REVIEW

File: linkis-public-enhancements/linkis-datasource/datasourcemanager/server/src/main/java/com/webank/wedatasphere/linkis/datasourcemanager/core/restful/DataSourceAdminRestfulApi.java [hunk 1]
```diff
@@ -63,148 +63,122 @@ public void initRestful(){
...
+        if(null == storedDataSourceEnv){
+            return Message.error("Fail to update data source environment[更新数据源环境失败], " + "[Please check the id:'"
+                    + envId + " is correct ']");
+        }
+        dataSourceEnv.setCreateUser(storedDataSourceEnv.getCreateUser());
+        updateDataSourceEnv(dataSourceEnv, storedDataSourceEnv);
+        return Message.ok().data("update_id", envId);
+    }
+
+    @RequestMapping(value = "/env/{env_id}/form",method = RequestMethod.PUT,headers = MediaType.MULTIPART_FORM_DATA)
```
Comment (casionone) [inline linkis-public-enhancements/linkis-datasource/datasourcemanager/server/src/main/java/com/webank/wedatasphere/linkis/datasourcemanager/core/restful/DataSourceAdminRestfulApi.java:147]: heders or consumes? 
javas.ws.rs.core.MediaType can replace by org.springframework.http.MediaType

File: linkis-public-enhancements/linkis-datasource/datasourcemanager/server/src/main/java/com/webank/wedatasphere/linkis/datasourcemanager/core/restful/DataSourceAdminRestfulApi.java [hunk 2]
```diff
@@ -63,148 +63,122 @@ public void initRestful(){
         this.formDataTransformer = FormDataTransformerFactory.buildCustom();
     }
 
-    @POST
-    @Path("/env/json")
-    public Response insertJsonEnv(DataSourceEnv dataSourceEnv, @Context HttpServletRequest req){
-        return RestfulApiHelper.doAndResponse(()->{
```
Comment (casionone) [inline linkis-public-enhancements/linkis-datasource/datasourcemanager/server/src/main/java/com/webank/wedatasphere/linkis/datasourcemanager/core/restful/DataSourceAdminRestfulApi.java:69]: About RestfulApiHelper.doAndResponse 
may be modfify  RestfulApiHelper.doAndResponse return  object of Response to Message will be better

**Comment by casionone** [comment]
> ### What is the purpose of the change
> [module of linkis-datasource] Replace jersey's restful style with spring web
> Related issues: #993.

**Review by casionone — APPROVED** [review]

**Review by casionone — APPROVED** [review]

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**